### DISCUSSION TITLE
My screens "break" for no reason

### DISCUSSION THREAD

**Discussion by sb-production** [outline/outline#5336] [Self Hosting]
I use copy/paste to insert my images. After a while, some of them "break" (see image)... Do you know where this could come from?

![2023-05-11_14-55-21](https://github.com/outline/outline/assets/129378921/75ff9e2a-514f-4ed2-b5a2-292a9e432c9c)

**Comment by tommoor** [comment]
What's the associated network error?

**Reply by sb-production** [reply]
There is no error indicated... Or I am not looking in the right place...
On the other hand I see that the path is not the same between an image that is ok : 
![image](https://github.com/outline/outline/assets/129378921/edf9ba86-f9aa-4036-816e-e0f8f8901d12)

and one that breaks: 
![image](https://github.com/outline/outline/assets/129378921/987c67f5-7f39-4c15-a0a1-e2bc2f465a5c)

**Reply by tommoor** [reply]
The network error will be in the developer tools console.

**Reply by sb-production** [reply]
![image](https://github.com/outline/outline/assets/129378921/22688f80-2eda-4c69-a600-87f075bee160)

**Reply by sb-production** [reply]
@tommoor Sorry for the delay

**Reply by tommoor** [reply]
I don't understand french, but it's a websocket error so wouldn't be related. You should check if the image still exists in your storage
